storefront-product-sharing.php

```diff
@@ -215,12 +215,6 @@ public function sps_install_storefront_notice() {
 	 * @return  void
 	 */
 	public function sps_styles() {
-		global $storefront_version;
-
-		if ( version_compare( $storefront_version, '2.3.0', '>=' ) ) {
```

Comment:
I'm not aware of a documented/published version support policy for storefront and the icons have only available from storefront 2.5.7 so might be safer to simply adjust this logic to be >= 2.30 and <= 2.5.6 so it m

Reply:
@luislard ^ just in case you missed it